platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "display_server_x11.h"
+#include <X11/Xlib.h>
```

Comment:
Already included in the `display_server_x11.h`.

Reply:
My Youcompleteplugin did that. I didn't realize it. My bad.